lib/resources/example_json/r4_examples_allergy_intolerance.rb

```diff
@@ -113,5 +113,78 @@ module Resources
         "resource": R4_ALLERGY_INTOLERANCE_ENTRY
       ]
     }
+
+    R4_ALLERGY_INTOLERANCE_CREATE ||= {
+      "resourceType": "AllergyIntolerance",
+      "contained":[
+        {
+          "resourceType": "PractitionerRole",
+          "id": "638995",
+          "code":[
+            {
+              "coding":[
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/practitioner-role",
```

Comment:
update to secured endpoint

Reply:
Interesting, may i know if there is a particular reason we would like to get these updated to https endpoints? (Based on your comment here https://github.com/cerner/fhir.cerner.com/pull/324#pullrequestreview-320097210)

All the systems are as defined at https://www.hl7.org/fhir/allergyintolerance.html with a http.

Reply:
yeah might be good idea to keep it secured. Because it will show as Not Secured warning when you click the link. If you missed this comment from earlier- _In general please go through the URL's and update them to secured_

Reply:
talked offline and with other team members and decided to keep system url's as http.